docs/setting-up.md

```diff
@@ -75,3 +74,5 @@ We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA (exp
 Support requests related to other IDEs will not be entertained.
 
 Refer to [this document](ide-usage.md) if you wish to set up an IDE for developing TEAMMATES.
+
+If you do not wish to set up an IDE, then you may proceed to the development routine as outlined in [this document](development.md).
```

Comment:
This should be listed as a separate step (step 5) since step 4 is marked as optional:

## Step 5: Start developing

If you followed every step correctly, you should have successfully set up the development environment. Proceed to the development routine as outlined in [this document](development.md).